src/index.html

```diff
@@ -1,13 +1,20 @@
 <!--
 
-    Copyright (c) 2015 Codenvy, S.A.
```

Comment:
I would leave this file  src/index.html untouched as it's only style that is not related to the issue